oxidation/libparsec/crates/platform_local_db/src/executor.rs

```diff
@@ -200,17 +199,22 @@ impl BackgroundSqliteExecutor {
                 Operation::Job(job) => job(&mut connection),
             }
         }
+        log::info!("BackgroundSqliteExecutor finished all his jobs");
     }
 }
 
 #[cfg(test)]
 mod tests {
     use crate::DatabaseError;
     use diesel::{connection::SimpleConnection, Connection, SqliteConnection};
+    use rstest::rstest;
+    use std::time::Duration;
 
     use super::SqliteExecutor;
 
-    #[tokio::test]
+    #[rstest]
+    #[timeout(Duration::new(60, 0))]
```

Comment:
```suggestion
    #[timeout(Duration::new(30, 0))]
```

`30sec` to `open -> close -> re-open` a file should be plenty enough

Reply:
Done in another PR